tests/utils/frame.test.ts

```diff
@@ -19,6 +19,18 @@ describe('frame tests', () => {
         expect(Frame.arrayStuff(unstuffedBytes)).toStrictEqual(stuffedBytes);
     });
 
+    test('test stuff/unstuff with 7 data bits wide', () => {
+        const unstuffedBytes = getBytesFromHex(
```

Comment:
it can be done in one line as it's rather short

Reply:
fixed